aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,67 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
+
+ The follwoing instrcutions were tested using CentOS 7.
+ 
+Install Mono
+^^^^^^^^^^^^
 
 .. note::
 
-    Installation steps for CentOS, Fedora and derivatives are not currently available but should be available soon. The commands are mostly the same, with some differences to account for the different package managers used on these systems. Learn how you can `contribute <https://github.com/aspnet/Docs/blob/master/CONTRIBUTING.md>`_ on GitHub.
-  
+This step it's optional if you want only use CoreCLR.
+
+`Mono <http://mono-project.com>`_ is an ongoing effort to port the .NET Framework to other platforms. Mono is one of the ways .NET applications can run on platforms other than Windows. ASP.NET 5 requires a version of Mono greater than 4.0.1.
+
+To install Mono::
+
+First import the public key from ubuntu.com::
+
+    sudo rpm --import "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF"
+
+Now your CentOS 7 have the public key from this server.
+
+In the next Step we want add the repo from the mono project. For this step we need a tool collection called yum-utils.
```

Comment:
This would be a little clearer as "Now, we have to add the yum repository provided by the Mono project. For this step we need to install the `yum-utils` package"